crates/isograph_lang_parser/src/parse_iso_literal.rs

```diff
@@ -252,6 +255,7 @@ fn parse_client_pointer_declaration_inner(
         Ok(ClientPointerDeclaration {
             parent_type,
```

Comment:
let's add `parse_directives` above. Otherwise, you get `selection sets are required` if you accidentally add a directive. Anyway, directives will be eventually supported so for now, we should either parse them and throw if they're present (most preferred) or silently ignore them (least preferred) or add them as an unused field on client pointer

could be separate PR